lib/ssh/src/ssh.erl

```diff
@@ -72,7 +72,7 @@ The descriptions of the options uses the
 > #### Note {: .info }
 >
 > The [User's Guide](index.html) has examples and a
```

Comment:
```suggestion
> The [User's Guide](index.html) has examples and an
```

Reply:
this sentence does not make sense. i will try to improve it.